### PR TITLE
Retrieve internal links from nested blocks

### PR DESCRIPTION
**Pull request by davisagli**
Handle retrieving internal links for linkintegrity checks from the same locations in block data that are supported by the resolveuid transform.

### CODE REVIEW

**Comment by davisagli** [comment]
@jenkins-plone-org please run jobs

**Comment by davisagli** [comment]
@jenkins-plone-org please run jobs

**Comment by davisagli** [comment]
@jenkins-plone-org please run jobs

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -6,15 +6,17 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python: [3.7, 3.8, 3.9, "3.10"]
-        plone: ["6.0.0b3", "5.2.9"]
+        python: [3.7, 3.8, 3.9, "3.10", "3.11"]
```
Comment (tisto) [inline .github/workflows/tests.yml:9]: @davisagli I guess we should reflect this in the changelog as well as in setup.py, what do you think?

**Review by tisto — APPROVED** [review]

File: .github/workflows/tests.yml (hunk 1 above)
Comment (tisto) [reply .github/workflows/tests.yml:9]: @davisagli ignore my comment. Seems that @mauritsvanrees added the classifiers for Python 3.10 and 3.11 some time ago already. We just did not test 3.11 properly so far. Thanks for fixing this!

**Pull request merged**